tests/test_clusters/test_cluster.py

```diff
@@ -141,5 +158,61 @@ def test_byo_proxy(byo_cpu, local_folder):
     # assert "sample_file_0.txt" in local_folder.ls(full_paths=False)
 
 
+@pytest.mark.clustertest
+def test_cluster_with_den_auth(ondemand_https_cluster_with_auth, summer_func_with_auth):
+    ondemand_https_cluster_with_auth.restart_server()
+    from runhouse.globals import configs
+
+    # Create an invalid token, confirm the server does not accept the request
+    orig_token = configs.get("token")
+
+    # Request should return 200 using a valid token
+    summer_func_with_auth(1, 2)
+
+    configs.set("token", "abcd123")
+
+    try:
```

Comment:
We need to try calling a bunch of other cluster methods with a bad token too, not just a function call